problem-12.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+// Takes a number as input and determines whether it is an Armstrong number or not
+
+int exponent(int number, int power)
+{
+    int exp = 1;
+    for (int i = 0; i < power; i++)
+    {
+        exp *= number;
+    }
+    return exp;
+}
+
+int main()
+{
+    int number, sum = 0, numberOfDigits = 0;

```

Comment:
`digitsCount` instead of `numberOfDigits`?  because the current one is a bit confusing.